src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/dm/DocumentManagementConfig.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.dm;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+public class DocumentManagementConfig {
+
+    @Bean
+    public RestTemplate restTemplate() {
+        return new RestTemplate();
+    }
```

Comment:
is this related to datetime change?

Reply:
no, it is not related to DateTime change but this will fix the master build, which is failing atm